server.go

```diff
@@ -87,34 +115,10 @@ func (s *Server) Context() context.Context {
 // Serve the grid on the listener. The listener address type must be
 // net.TCPAddr, otherwise an error will be returned.
 func (s *Server) Serve(lis net.Listener) error {
-	// Create a registry client, through which other
-	// entities like peers, actors, and mailboxes
-	// will be discovered.
-	r, err := registry.New(s.etcd)
-	if err != nil {
-		return err
-	}
-	s.registry = r
 	s.registry.Timeout = s.cfg.Timeout
```

Comment:
If we're creating the client in `NewServer()` we might as well move the timeout and lease duration setting there as well for clarity.